common/djangoapps/edeos/models.py

```diff
@@ -8,14 +8,16 @@ class UserSocialLink(models.Model):
     """
     Represents a user's social platforms links.
     """
-    LINKEDIN = 'linkedin'
+    # LINKEDIN = 'linkedin'
+    FACEBOOK = 'facebook'
 
     PLATFORM_CHOICES = (
-        (LINKEDIN, ugettext_noop('LinkedIn')),
+        # (LINKEDIN, ugettext_noop('LinkedIn')),
+        (FACEBOOK, ugettext_noop('Facebook')),
     )
 
     VALID_URLS = {
-        LINKEDIN: 'linkedin.com/in/'
+        FACEBOOK: 'Facebook.com/in/'
```

Comment:
Let's put this PR on hold (discussed with Ruslan). 
We need to change the flow of social links verification a bit (replace LinkedIn with Facebook), it's not a task to just change the text

Reply:
@RuslanSkiraRaccoongang please update the verification link https://raccoongang.slack.com/archives/CDQ7E16RX/p1560253617115200

Reply:
I checked it with _Facebook.com_ not working so I use _Facebook.com/_